### ISSUE TITLE
poor performance on CIFAR10

### ISSUE THREAD

**Issue by Peony-Wang** [ricky40403/DSQ#3]
Hi, thank you for your implementation!

I use the CIFAR10 dataset for quick training and evaluation on the 8-bit setting, but I only got not more than 80%@Top1 accuracies with/without  activation quantization. Could you  give me any clue of why this happens?  The learning rate is decreased by 10x every 30 epochs. But the accuracy stops increasing after around 40-50 epochs. 

Thanks for your attention!

**Comment by ricky40403** [comment]
Hi Peony, sorry that I did not test on CIFAR10. Did you test that training without quantization? And what about the initial learning rate?
